dynamic_instrumentation/src/instrument_memory.rs

```diff
@@ -231,12 +252,12 @@ impl<'a, 'tcx: 'a> Visitor<'tcx> for FunctionInstrumenter<'a, 'tcx> {
                         let field_fn = self
                             .find_instrumentation_def(Symbol::intern("ptr_field"))
                             .expect("Could not find pointer field hook");
-                        self.add_instrumentation(
+                        self.add_instrumentation_point(
                             location,
                             field_fn,
                             vec![
-                                Operand::Copy(base.local.into()),
-                                make_const(self.tcx, field.as_u32()),
+                                InstrumentationOperand::Place(Operand::Copy(base.local.into())),
```

Comment:
most of these are already confirmed to be unsafe pointer types, so I think those cases should be InstrumentationOperand::Raw

Reply:
You're right, I audited these sites and commented on each of them in #468.